test/e2e/environment/project_test.go

```diff
@@ -58,6 +60,41 @@ func TestListProjects(t *testing.T) {
 	}
 }
 
+func TestListProjectsRequestOrganizations(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+	c := newEnvironmentClient(t)
+	defer c.Close()
+	pageSize := int64(1)
```

Comment:
Because we want to test two or more organizations, we must set the `pageSize` above 2.

Reply:
Fixed to get two organization IDs.
https://github.com/bucketeer-io/bucketeer/pull/810/commits/ada5e82f03bc3759142de7ed354bfb57e7b99a63